AI-and-Analytics/Getting-Started-Samples/LPOT-Sample-for-Tensorflow/README.md

```diff
@@ -47,20 +47,33 @@ We will learn how to train a CNN model based on Keras with Tensorflow, use LPOT
 | OS                                | Linux* Ubuntu* 18.04 
 | Hardware                          | The Second Generation Intel(R) Xeon(R) Scalable processor family or newer
 | Software                          | Intel(R) oneAPI AI Analytics Toolkit 2021.1 or newer
-| What you will learn               | How to use LPOT tool to quantize the AI model based on Tensorflow and speed up the inference on Intel(R) Xeon(R) CPU
+| What you will learn               | How to use LPOT tool to quantize the AI model based on TensorFlow and speed up the inference on Intel(R) Xeon(R) CPU
 | Time to complete                  | 10 minutes
 
 ## Running Environment
 
-### Running in Devcloud
+### Running Samples In DevCloud (Optional)
 
-If running a sample in the Intel DevCloud, please follow the below steps to build the python environment. Also, remember that you must specify the compute node (CPU) as well as whether to run in batch or interactive mode. For more information, see the [Intel(R) oneAPI AI Analytics Toolkit Get Started Guide](https://devcloud.intel.com/oneapi/get-started/analytics-toolkit/)
+### Run in Interactive Mode
```

Comment:
Those are general contents for all samples.
Could you move out the contents from specific sample's README and move it to a general place such as README for all AI samples?

ex: https://github.com/oneapi-src/oneAPI-samples/blob/master/AI-and-Analytics/README.md

Reply:
@louie-tsai  DX (Donna) says that user studies show that users are not looking at the generic readme so we need to have the instructions in the README for each sample. 
I will also add the generic instructions to the AI General README, but I don't want to remove them from the README for each sample.